bin

```diff
@@ -36,7 +59,7 @@ app.use(function(req, res) {
         },
         plugins: [
             new webpack.DefinePlugin({
-                REQUIRE_PATH: JSON.stringify("." + path),
+                REQUIRE_PATH: JSON.stringify(requirePath),
```

Comment:
Sadly, I'm getting lots of "module not found" errors when I attempt to `require()` this from `entry.js`.